packages/frontend/src/components/PageLayout/PageLayout.tsx

```diff
@@ -90,13 +89,16 @@ export const ProtectedPageLayout = () => {
 export const PageLayout: React.FC = () => {
   const queryClient = useQueryClient();
   const { t } = useTranslation();
-  const { selectedParties, allOrganizationsSelected } = useParties();
+  const { selectedParties, allOrganizationsSelected, isCompanyFromParams } = useParties();
   const [searchParams] = useSearchParams();
 
   useProfile();
 
   const name = allOrganizationsSelected ? t('parties.labels.all_organizations') : selectedParties?.[0]?.name || '';
-  const isCompany = allOrganizationsSelected || selectedParties?.[0]?.partyType === 'Organization';
+
+  const isCompany =
```

Comment:
Maybe `useParties` should calculate and expose `currentProfile` instead? Either `company` or `person`.